src/SportsHub.Business/Services/Implementation/SubcategoryService.cs

```diff
@@ -41,5 +41,10 @@ public async Task<Guid> FindSubcategoryIdBySubcategoryNameAsync(string subcatego
         {
             return await _subcategoryRepository.FindSubcategoryIdBySubcategoryNameAsync(subcategoryName);
         }
+
+        public IQueryable<Subcategory> GetSubcategoryIdByCategoryIdAsync(Guid categoryId)
```

Comment:
It's not async
Why do you return IQueryable?
You return subcategory, but the name says that you return subcategory id
```suggestion
        public Tack<IEnumerable<Subcategory>> GetByCategoryIdAsync(Guid categoryId)
```

Reply:
I'm sorry, my bad, I misunderstood the previous comments. I'll fix this
I'll make this method async
I forgot to rename this method after previous changes, so I'll rename